scripts/release.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+## Usage scripts/release.sh blogpost.txt <prerelease/patch/minor/major>
+# The first argument blogpost.txt is published to the jbrowse 2 blog. The
+# second argument is a flag for the publishing command for version bump, by
+# default it is none and will prompt
+
+## Precautionary bash tags
+set -e
+set -u
+set -o pipefail
+set -x
+
+NOTES=`cat $1`
+DATE=$(date +"%Y-%m-%d")
+BLOGPOST_FILENAME=products/website/blog/$(date +"%Y-%m-%d")-release.md
+
+yarn run lerna-publish $2
```

Comment:
Since `lerna-publish` uses several commands chained with `&&`, I think this will actually pass `$2` to the last command in the chain, not the first. Maybe the "push web tag, push other tags" steps could be part of this script?

Reply:
At one point I think I was opting to move that to a postlerna-publish step but it's probably good to move to this script

Reply:
Moved to the script :)